### PR TITLE
refactor(interactions): use the new `channel` field

### PR DESCRIPTION
**Pull request by TTtie**
Ref: https://github.com/discord/discord-api-docs/commit/88a761816b26d6891f8a276683bf57a4133aa233

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -69,6 +69,7 @@ declare namespace Dysnomia {
   type GuildTextableChannel = TextChannel | TextVoiceChannel | NewsChannel | StageChannel;
   type GuildTextableWithThreads = GuildTextableChannel | AnyThreadChannel;
   type InviteChannel = InvitePartialChannel | Exclude<AnyGuildChannel, CategoryChannel | AnyThreadChannel>;
+  type PossiblyUncachedInteractionChannel = Textable | { id: string; type: ChannelTypes };
```
Comment (coolcalcacol) [inline index.d.ts]: Maybe use `Pick<Channel, 'id' | 'type'>`?

or use a new interface on line 63?
  ```interface UncachedChannel extends Uncached { type: ChannelTypes }```

Comment (coolcalcacol) [reply index.d.ts]: Textable also doesn't have any channel related properties, is there a better type we can use?

**Review by coolcalcacol — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (TTtie) [reply index.d.ts]: Maybe `TextableChannel | PartialChannel` would be a better choice?

Comment (coolcalcacol) [reply index.d.ts]: Yeah, I agree.

Also, type in PartialChannel is a number instead of ChannelTypes. Something to update here?

Comment (TTtie) [reply index.d.ts]: Definitely something to tackle (in a different PR)

**Pull request merged**